blocks/learning-techniques/learning-techniques.css

```diff
@@ -0,0 +1,9 @@
+.learning-techniques{
+    background-color: #fff;
```

Comment:
The property `background-color: #fff;` is not necessary here and for other blocks, as the default background color of the page is white.